app/src/main/res/values-es/strings.xml

```diff
@@ -30,6 +30,9 @@
     <string name="all_cancel">Cancelar</string>
     <string name="all_confirmation_question">¿Estás seguro?</string>
 
+    <string name="logout_title">Ups! No puedes cerrar sesión</string>
+    <string name="logout_description">Por favor asegurate de tener conexión a Internet y vuelve a intentarlo más tarde</string>
```

Comment:
Add  "Tienes snapshots pendientes de envío".  Alert is about pending snapshots, not precisely because lack of Internet.

Reply:
@alefq I agreed 100%. The message was chosen by the guys at FP. The idea behind is that the user not necessarily knows what a snapshot even is or how to sync them.